app/models/course.rb

```diff
@@ -268,9 +270,19 @@ def notify_accredited_body?
     (saved_changes.keys & update_notification_attributes).any?
   end
 
-  def send_notification_to_accredited_body
-    users = UserNotification.where(provider_code: accrediting_provider_code, course_update: true).map(&:user)
+  def send_create_notification_to_accredited_body
+    users = User.joins(:user_notifications).merge(UserNotification.course_create_notification_requests(accrediting_provider_code))
```

Comment:
This looks a bit hairy but it avoids the `n+1` of using `map`. We haven't used query objects in the app as yet but this might be a good candidate for one. I don't think we should do it now though, just a consideration for when we refactor this.